src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,280 @@
+		await game.gameMessage.reactions.removeAll();
+
+		const whatToDo = reactionMap.size === 0
+			? EndingAction.TIMEOUT
+			: reactionMap.first()!.emoji.name === ReactionEmoji.OK
+				? EndingAction.PLAY
+				: EndingAction.STOP;
+
+		// Decide whether we timeout, stop, or continue
+		switch (whatToDo) {
+			case EndingAction.TIMEOUT:
+				await game.gameMessage.edit(language.tget('COMMAND_HIGHERLOWER_TIMEOUT'), { embed: null });
+			case EndingAction.STOP:
+				await this.end(game, message, true);
+				break;
+			case EndingAction.PLAY:
+				await game.gameMessage.edit(language.tget('COMMAND_HIGHERLOWER_NEWROUND'), { embed: null });
+				break;
+		}
+		return whatToDo === EndingAction.PLAY;
+	}
+
+	private async loss(game: HigherLowerGameData, message: KlasaMessage) {
+		// TODO (Quantum): Implement losing event to InfluxDB
+		const { number, wager, turn, gameMessage } = game;
+		await gameMessage.reactions.removeAll();
+		let losses = wager;
+
+		// There's a 0.1% chance that a user would lose now only the wager, but also what they would've won. Muahaha
+		if ((Math.random()) < 0.001) {
+			losses += this.calculateWinnings(wager, turn - 1);
+			await message.author.settings.decrease(UserSettings.Money, losses);
+		}
+
+
+		const { TITLE, DESCRIPTION, FOOTER } = message.language.tget('COMMAND_HIGHERLOWER_LOSE');
+		await game.gameMessage.edit({
+			content: null,
+			embed: new MessageEmbed()
+				.setColor(getColor(message))
+				.setTitle(TITLE)
+				.setDescription(DESCRIPTION(number, losses))
+				.setFooter(FOOTER)
+		});
+		return false;
+	}
+
+	private async end(game: HigherLowerGameData, message: KlasaMessage, cashout?: boolean) {
+		// Remove all the reactions and end the LLRC
+		await game.gameMessage.reactions.removeAll();
+		game.llrc!.end();
+
+		// Should we need to cash out, proceed to doing that
+		if (cashout!) {
+			const { turn, wager } = game;
+			// Let the user know, and nullify the embed
+			await game.gameMessage.edit(message.language.tget('COMMAND_HIGHERLOWER_CASHOUT_INIT'), { embed: null });
+
+			// Calculate and deposit winnings for that game
+			const winnings = this.calculateWinnings(wager, turn - 1);
+			await message.author.settings.increase(UserSettings.Money, winnings);
+
+			// Let the user know we're done!
+			await game.gameMessage.edit(message.language.tget('COMMAND_HIGHERLOWER_CASHOUT', winnings));
+			return;
+		}
+
+		// Say bye!
+		const { TITLE, DESCRIPTION } = message.language.tget('COMMAND_HIGHERLOWER_CANCEL');
+		await game.gameMessage.edit({
+			content: null,
+			embed: new MessageEmbed()
+				.setColor(getColor(message))
+				.setTitle(TITLE)
+				.setDescription(DESCRIPTION(message.author.username))
+		});
+	}
+
+	private async reactionCollector(ctx: EmojiPromiseContext, message: KlasaMessage, game: HigherLowerGameData, reaction: LLRCData) {
+		// Ignore if resolve is not ready
+		if (typeof ctx.resolve !== 'function'
```

Comment:
```suggestion
		if (!isFunction(ctx.resolve
```
And then `import { isFunction } from '@klasa/utils';` at the top of the file

Reply:
Diffs for the other files so you can pretty much copypasta

<details><summary><code>hungergames.ts</code></summary>
<p>

```diff

diff --git a/src/commands/Games/hungergames.ts b/src/commands/Games/hungergames.ts
index aa4793e4..08bc00f6 100644
--- a/src/commands/Games/hungergames.ts
+++ b/src/commands/Games/hungergames.ts
@@ -1,10 +1,11 @@
+import { isFunction } from '@klasa/utils';
 import { CommandStore, KlasaMessage, Language, util } from 'klasa';
 import { SkyraCommand } from '../../lib/structures/SkyraCommand';
 import { Events } from '../../lib/types/Enums';
+import { GuildSettings } from '../../lib/types/settings/GuildSettings';
 import { HungerGamesUsage } from '../../lib/util/Games/HungerGamesUsage';
 import { LLRCData, LongLivingReactionCollector } from '../../lib/util/LongLivingReactionCollector';
 import { cleanMentions } from '../../lib/util/util';
-import { GuildSettings } from '../../lib/types/settings/GuildSettings';

 const EMOJIS = ['🇳', '🇾'];

@@ -52,13 +53,13 @@ export default class extends SkyraCommand {
                        bloodbath: true,
                        llrc: new LongLivingReactionCollector(this.client, async reaction => {
                                // Ignore if resolve is not ready
-                               if (typeof resolve !== 'function'
+                               if (!isFunction(resolve)
                                // Run the collector inhibitor
                                || await this.collectorInhibitor(message, gameMessage!, reaction)) return;
                                resolve(Boolean(EMOJIS.indexOf(reaction.emoji.name)));
                                resolve = null;
                        }, () => {
-                               if (typeof resolve === 'function') resolve(false);
+                               if (isFunction(resolve)) resolve(false);
                                this.playing.delete(message.channel.id);
                        }),
                        sun: true,
```

</p>
</details>

<details><summary><code>Limiter.ts</code></summary>
<p>

```diff
diff --git a/src/lib/util/Limiter.ts b/src/lib/util/Limiter.ts
index 626d70f6..37292d3a 100644
--- a/src/lib/util/Limiter.ts
+++ b/src/lib/util/Limiter.ts
@@ -1,8 +1,9 @@
-import { createMethodDecorator } from 'klasa-decorators';
+import assert from 'assert';
 import { Collection, Constructor } from 'discord.js';
 import { RateLimit } from 'klasa';
+import { createMethodDecorator } from 'klasa-decorators';
+import { isFunction } from 'util';
 import { createClassDecorator } from './util';
-import assert from 'assert';

 export const enum LimitErrors {
        Ratelimited = 'RATELIMIT'
@@ -37,7 +38,7 @@ export function limitMethod(group = 'global', bucket: number, cooldown: number)
                const method = descriptor.value;

                if (!method) throw new Error('Function limiter actions require a [[value]].');
-               if (typeof method !== 'function') throw new Error('Function limiter actions can only be applied to functions.');
+               if (!isFunction(method)) throw new Error('Function limiter actions can only be applied to functions.');

                descriptor.value = (function descriptorValue(this: LimitedClass, ...args: readonly unknown[]) {
                        assert(typeof this.limiter === 'boolean');
```

</p>
</details>

<details><summary><code>commands.ts</code></summary>
<p>

```diff
diff --git a/src/routes/commands.ts b/src/routes/commands.ts
index abd3ee90..800174c6 100644
--- a/src/routes/commands.ts
+++ b/src/routes/commands.ts
@@ -1,3 +1,4 @@
+import { isFunction } from '@klasa/utils';
 import { Route, RouteStore } from 'klasa-dashboard-hooks';
 import ApiRequest from '../lib/structures/api/ApiRequest';
 import ApiResponse from '../lib/structures/api/ApiResponse';
@@ -22,7 +23,7 @@ export default class extends Route {
                        bucket: cmd.bucket,
                        category: cmd.category,
                        cooldown: cmd.cooldown,
-                       description: typeof cmd.description === 'function' ? cmd.description(language) : cmd.description,
+                       description: isFunction(cmd.description) ? cmd.description(language) : cmd.description,
                        guarded: cmd.guarded,
                        guildOnly: !cmd.runIn.includes('dm'),
```

</p>
</details>